src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -182,22 +184,28 @@ public CourseDetailsBundle getCourseDetails(String courseId) throws EntityDoesNo
     }
 
     public List<String> getSectionsNameForCourse(String courseId) throws EntityDoesNotExistException {
+        return getSectionsNameForCourse(courseId, true);
+    }
 
-        verifyCourseIsPresent(courseId);
-        
+    private List<String> getSectionsNameForCourse(
+        String courseId, boolean checkExists) throws EntityDoesNotExistException {
```

Comment:
Not sure. I don't know the meaning of that parameter. :-) May be you can add a comment for the method?